supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -1,17 +1,19 @@
 [[support]]
 = Support
 
-[discrete]
+[[technology-preview]]
```

Comment:
This ID needs to be changed to a unique value. Since this ID is already in use elsewhere in `t.adoc`, when I build, I get the following asciidoctor error:

`asciidoctor: WARNING: glossary_terms_conventions/general_conventions/t.adoc: line 50: id assigned to block already in use: technology-preview`

Reply:
Good catch @bburt-rh! Could update to something like [[technology-preview-guidance]] @leswilliams44

Reply:
@bburt-rh and @bergerhoffer, I revised the ID.

Reply:
Is there one last item? I am not sure what it is.

Reply:
Thanks @leswilliams44. This now builds correctly and lgtm.